protzilla/importing/ms_data_import.py

```diff
@@ -1,14 +1,18 @@
 import logging
 import re
+import traceback
 from collections import defaultdict
 
 import numpy as np
 import pandas as pd
 
 from protzilla.data_integration.database_query import biomart_query
+from protzilla.utilities import format_trace
 
 
-def max_quant_import(file_path: str, intensity_name: str, map_to_uniprot=False) -> dict:
+def max_quant_import(
+    file_path: str, intensity_name: str, map_to_uniprot=False, aggregation_method="Sum"
+) -> dict:
```

Comment:
Why do we pass the aggregation method if we don't use it?
If we use it later on it maybe makes more sense to specify it there?